save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/ContainerLogController.kt

```diff
@@ -0,0 +1,82 @@
+package com.saveourtool.save.backend.controllers
+
+import com.saveourtool.save.backend.service.AgentService
+import com.saveourtool.save.configs.ApiSwaggerSupport
+import com.saveourtool.save.service.LogService
+import com.saveourtool.save.utils.blockingToMono
+import com.saveourtool.save.v1
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.Parameter
+import io.swagger.v3.oas.annotations.Parameters
+import io.swagger.v3.oas.annotations.enums.ParameterIn
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.tags.Tag
+import io.swagger.v3.oas.annotations.tags.Tags
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import reactor.core.publisher.Flux
+import java.time.Instant
+import java.time.ZoneId
+
+typealias StringFluxResponse = ResponseEntity<Flux<String>>
+
+/**
+ * Controller to fetch logs
+ */
+@ApiSwaggerSupport
+@Tags(
+    Tag(name = "logs"),
+)
+@RestController
+@RequestMapping(path = ["/api/$v1/logs"])
+class ContainerLogController(
+    private val logService: LogService,
+    private val agentService: AgentService,
+) {
+    @GetMapping("/by-container-name")
+    @Operation(
+        method = "GET",
+        summary = "Get logs for containerName in provided time range.",
+        description = "Get logs for containerName in provided time range.",
+    )
+    @Parameters(
+        Parameter(name = "containerName", `in` = ParameterIn.QUERY, description = "name of a container", required = true),
+        Parameter(name = "from", `in` = ParameterIn.QUERY, description = "start of requested time range", required = true),
```

Comment:
need to test how Spring parses dates.
also, need to add an additional endpoint to return current logs (from = now - delta, to = now)

Reply:
it parsed properly from ISO format. will be enhanced after adding a FE part

the second method also will be added in scope of CPG logs